Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM cibuilds/hugo:0.80 AS builder
+
+COPY content content
+COPY static static
+COPY themes themes
+COPY resources resources
+COPY config.toml .
```

Comment:
This looks good to me. As long as we don't expect any new directories showing up? 

could we also do something like:
```
COPY ./ /yt-blog
WORKDIR /yt-blog
```

Or would that  affect the copy later?